src/global/Constants.h

```diff
@@ -51,3 +52,4 @@ static const int DEFAULT_NOF_DATE_YEAR_DIGITS = 19;
 // How many lines are parsed at once during index creation.
 // Reduce to save RAM
 static const int NUM_TRIPLES_PER_PARTIAL_VOCAB = 100000000;
+static const std::string PARTIAL_VOCAB_FILE_NAME = ".partial_vocabulary";
```

Comment:
Use `-`instead of `_` to make even more consistent. I think I misstated it for snake case in the last review.